README.md

```diff
@@ -1,10 +1,21 @@
-## Welcome to GitHub Pages
+## Welcome to Mateusz' Awesome Software Page
+
+### Introduction
+
+This is some information here.
+
+### Contribution
+
+This is how to contribute. Use of **PR** is strongly recommended! :)
```

Comment:
Please don't use bold in the text. Can you change it to italics?

Reply:
ok, sure! Should be ok now @mkuzak :)